app/controllers/collections_controller.rb

```diff
@@ -23,4 +23,15 @@ def create
     redirect_to solr_document_path(params[:apo_id]), notice: "Created collection #{collection_pid}"
     @apo.save # indexing happens automatically
   end
+
+  def exists
+    resp = if params[:title].present?
```

Comment:
Do we need to consider the case where title is present and unique and catkey is present and not unique?

Reply:
good question for Ben, but I'd guess a non-unique catkey should trigger a validation problem regardless of title uniqueness

Reply:
Because of how the form is constructed, you can only add one or the other.